autosklearn/ensembles/ensemble_selection.py

```diff
@@ -273,23 +286,25 @@ def _bagging(
             dtype=np.int64,
         )
 
-    def predict(self, predictions: Union[np.ndarray, List[np.ndarray]]) -> np.ndarray:
+    def predict(
+        self, base_models_predictions: Union[np.ndarray, List[np.ndarray]]
+    ) -> np.ndarray:
 
-        average = np.zeros_like(predictions[0], dtype=np.float64)
-        tmp_predictions = np.empty_like(predictions[0], dtype=np.float64)
+        average = np.zeros_like(base_models_predictions[0], dtype=np.float64)
+        tmp_predictions = np.empty_like(base_models_predictions[0], dtype=np.float64)
```

Comment:
In both cases `base_model_predictions` needs to be a 3D tensor. In the case of `List[np.ndarray]` it would be a list of 2d numpy arrays, and we would create arrays based on the shape of the first array. Similarly, in the case of `np.ndarray` it would be a 3d numpy array, and retrieving the first element would give us a 2d numpy array with the shape of one prediction.